src/pages/Lobby/Lobby.tsx

```diff
@@ -22,20 +23,25 @@ const Lobby = (): JSX.Element => {
 	const [lobby, setLobby] = useState<LobbyType>();
 	const [game, setGame] = useState<Game>();
 
+	const updateLobby = (lobby: LobbyType) => {
+		setLobby(lobby);
+		setGame(lobby?.game);
```

Comment:
Isn't it weird to have two states when one of the states is a subobject of a state?

Reply:
Actually yeah! Changing that.